tenzies/src/component/NumPad.jsx

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable react/prop-types */
+
+export default function NumPad(props) {
+  const styles={
+      backgroundColor: props.isHeld ? "#59E391" : "white",
+  }
+return (
+<button style={styles} className="num-button" onClick={()=>props.hold(props.id)}>
```

Comment:
in `props.hold(props.id)}` passing id is not needed, for two reasons, the first one that you don't pass an `id` to the component when you used the `NumPad` component in `tenzies/src/component/folder/Main.jsx`

the second reason is that you already passed hold an arrow function that has hard binding of the `id ` when you passed the `hold` prop like this in `Main.tsx`
```tsx
() => hold(num.id)
```

Usually typescript helps a lot in situations like this and will make you avoid any bugs related to types, as it will prevent you from calling the `hold` function with the wrong signature